src/mcts/search.h

```diff
@@ -117,14 +119,14 @@ class Search {
   // Sets stop to true and notifies watchdog thread.
   void FireStopInternal();
 
-  void SendMovesStats() const;
   // Function which runs in a separate thread and watches for time and
   // uci `stop` command;
   void WatchdogThread();
 
   // Populates the given list with allowed root moves.
-  // Returns true if the population came from tablebase.
-  bool PopulateRootMoveLimit(MoveList* root_moves) const;
+  // Returns best_rank != 0 if the population came from tablebase.
```

Comment:
What is best_rank? Where does it return it?

Reply:
In order to display true score when in syzygy filtered state, I changed the root probe code to return best rank or 0.